collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -210,7 +178,7 @@ public void onCreate(Bundle savedInstanceState) {
 
             MyDrive = false;
 
-            if (testNetwork()) {
+            if (isDeviceOnline()) {
```

Comment:
😢😢 this should just be `!isDeviceOnline()`. If you want to do a separate pull request to clean up some of these things that you may have seen while going through the code, feel free.